tests/integ/test_data_source_api.py

```diff
@@ -95,7 +93,8 @@
 ]
 
 SQL_SERVER_TABLE_NAME = "AllDataTypesTable"
-ORACLEDB_TABLE_NAME = "ALL_TYPES_TABLE"
+ORACLEDB_TABLE_NAME = "ALL_TYPE_TABLE"
+ORACLEDB_TABLE_NAME_SMALL = "ALL_TYPE_TABLE_SMALL"
```

Comment:
I'm thinking do we need to keep two tables? it feels redundant to me now.
what kind of tests we are using ALL_TYPE_TABLE_SMALL, and can we consolidate to use a single table?

Reply:
I remember I created a smaller table to test partition logic in different situation